crypto/cpu-intel.c

```diff
@@ -107,7 +107,7 @@ static void OPENSSL_cpuid(uint32_t *out_eax, uint32_t *out_ebx,
 //
 // See https://software.intel.com/en-us/articles/how-to-detect-new-instruction-support-in-the-4th-generation-intel-core-processor-family
 static uint64_t OPENSSL_xgetbv(uint32_t xcr) {
-#if defined(_MSC_VER) && !defined(__clang__)
+#if defined(_MSC_VER)
```

Comment:
When I started this work, I undid the manual prefixing by merging some files from BoringSSL to minimize the difference. This is one of the differences that was unrelated that was a result of that process. I decided to stop building this PR that way because there were too many of these unrelated changes; the work to minimize the diff from BoringSSL is now tracked in #1269.